packages/edit-navigation/src/components/menu-editor/use-navigation-blocks.js

```diff
@@ -89,7 +124,7 @@ export default function useNavigationBlocks( menuId ) {
 
 		// Disable reason, this code will eventually be implemented.
 		// eslint-disable-next-line no-unused-vars
-		for ( const clientId of deletedClientIds ) {
+		for ( const deletedClientId of deletedClientIds ) {
```

Comment:
If I do that, it fails the lint check because `clientId` is already declared in the upper scope. The deletion work will require some reworking to integrate with this PR anyway - which is what I attempted in #22128 😅